### PR TITLE
Updated docstrings

### PR DESCRIPTION
**Pull request by ceferisbarov**
Updated docstrings for:
- Rename
- Replace
- Coalesce
- Coerce
- Identity

### CODE REVIEW

File: src/transforms/coalesce.jl [hunk 1]
```diff
@@ -6,6 +6,16 @@
     Coalesce(value)
 
 Replaces all missing values from the table with `value`.
+
+# Examples
+
+```{julia}
```
Comment (juliohm) [inline src/transforms/coalesce.jl:12]:
```suggestion
```julia
```

File: src/transforms/coerce.jl [hunk 2]
```diff
@@ -9,9 +9,15 @@ Return a copy of the table, ensuring that the scientific types of the columns ma
...
 
+# Examples
+
 ```julia
 Coerce(:col1 => Continuous, :col2 => Count)
 ```
+
+## Notes
+
+* The transform can alter the element type of columns.
```
Comment (juliohm) [inline src/transforms/coerce.jl:20]:
```suggestion
```

File: src/transforms/identity.jl [hunk 3]
```diff
@@ -6,6 +6,14 @@
...
 
 The identity transform `x -> x`.
+
+# Examples
+
+T = Identity()
+
+## Notes
+
+* The schema of the table is preserved by the transform.
```
Comment (juliohm) [inline src/transforms/identity.jl:16]:
```suggestion
```

File: src/transforms/identity.jl [hunk 4]
```diff
@@ -6,6 +6,14 @@
     Identity()
 
 The identity transform `x -> x`.
+
+# Examples
+
+T = Identity()
```
Comment (juliohm) [inline src/transforms/identity.jl:12]: Missing the julia code block

File: src/transforms/rename.jl [hunk 5]
```diff
@@ -6,6 +6,17 @@
     Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
 
 The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+
+# Examples
+
+```{julia}
```
Comment (juliohm) [inline src/transforms/rename.jl:12]:
```suggestion
```julia
```

File: src/transforms/rename.jl [hunk 6]
```diff
@@ -6,6 +6,17 @@
...
+T = Rename(Dict(:a => :x, :c => :y))
+T = Rename("a" => "x", "c" => "y")
+```
+
+## Notes
+
+* The schema of the table is preserved by the transform.
```
Comment (juliohm) [inline src/transforms/rename.jl:19]:
```suggestion
```

File: src/transforms/replace.jl [hunk 7]
```diff
@@ -6,6 +6,17 @@
...
+# Examples
+
+```{julia}
+T = Replace(1 => -1, 5 => -5)
+T = Replace(1 => 1.5, 5 => 5.5, 4 => true)
+```
+
+## Notes
+
+* The transform can alter the element type of columns.
```
Comment (juliohm) [inline src/transforms/replace.jl:19]:
```suggestion
```

File: src/transforms/replace.jl [hunk 8]
```diff
@@ -6,6 +6,17 @@
     Replace(old₁ => new₁, old₂ => new₂, ..., oldₙ => newₙ)
 
 Replaces `oldᵢ` value with `newᵢ` value in the table.
+
+# Examples
+
+```{julia}
```
Comment (juliohm) [inline src/transforms/replace.jl:12]:
```suggestion
```julia
```

**Review by juliohm — CHANGES_REQUESTED** [review]

File: src/transforms/coalesce.jl (hunk 1 above)
Comment (eliascarv) [inline src/transforms/coalesce.jl:12]:
```suggestion
```julia
```

File: src/transforms/identity.jl (hunk 4 above)
Comment (eliascarv) [inline src/transforms/identity.jl:12]: Add 
\`\`\`julia 
#code
\`\`\`

File: src/transforms/rename.jl (hunk 5 above)
Comment (eliascarv) [inline src/transforms/rename.jl:12]:
```suggestion
```julia
```

File: src/transforms/replace.jl (hunk 8 above)
Comment (eliascarv) [inline src/transforms/replace.jl:12]:
```suggestion
```julia
```

File: src/transforms/rename.jl [hunk 9]
```diff
@@ -6,6 +6,17 @@
     Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
 
 The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+
+# Examples
+
+```julia
+T = Rename(Dict(:a => :x, :c => :y))
```
Comment (eliascarv) [inline src/transforms/rename.jl:13]: @juliohm, I think the `Rename(dict)` method is part of the internal API, should we expose this to the user?

Comment (juliohm) [reply src/transforms/rename.jl:13]: Good catch @eliascarv , I think we should omit it. Advanced users can always read the source. Agree?

**Comment by eliascarv** [comment]
```julia
T = Rename(Dict(:a => :x, :c => :y))
```
@juluihm, I think the `Rename(dict)` method is part of the internal API, should we expose this to the user?

File: src/transforms/rename.jl (hunk 9 above)
Comment (eliascarv) [inline src/transforms/rename.jl:13]:
```suggestion
```
This method is not part of public API.

File: src/transforms/coerce.jl [hunk 10]
```diff
@@ -9,9 +9,15 @@ Return a copy of the table, ensuring that the scientific types of the columns ma
 
 This transform wraps the ScientificTypes.coerce function. Please see their docstring for more details.
 
+# Examples
+
 ```julia
 Coerce(:col1 => Continuous, :col2 => Count)
```
Comment (eliascarv) [inline src/transforms/coerce.jl:15]:
```suggestion
T = Coerce(:col1 => Continuous, :col2 => Count)
```
Just for consistency.

**Comment by juliohm** [comment]
I was thinking that maybe we should erase the `T = ...` in front of every example. It doesn't help to name a variable in this context. Can you remove all occurrences of `T` in the docstrings @ceferisbarov ? Feel free to request a new review when the PR is ready for review again.

**Comment by eliascarv** [comment]
I agree, I will remove from the docstrings I made in the next PR.

**Comment by ceferisbarov** [comment]
So do we completely remove the docstring for `Rename`, or should we still leave some sort of comment explaining its behavior?

@juliohm @eliascarv

**Comment by juliohm** [comment]
I don't understand your question @ceferisbarov , why we would remove the docstring of Rename?

File: src/transforms/coerce.jl (hunk 2 above)
Comment (juliohm) [inline src/transforms/coerce.jl:20]:
```suggestion
```

File: src/transforms/identity.jl (hunk 3 above)
Comment (juliohm) [inline src/transforms/identity.jl:16]:
```suggestion
```

File: src/transforms/rename.jl [hunk 11]
```diff
@@ -6,6 +6,17 @@
...
+# Examples
+
+```julia
+T = Rename(:a => :x, :c => :y)
+T = Rename("a" => "x", "c" => "y")
+```
+
+## Notes
+
+* The schema of the table is preserved by the transform.
```
Comment (juliohm) [inline src/transforms/rename.jl:19]:
```suggestion
```

File: src/transforms/replace.jl [hunk 12]
```diff
@@ -6,6 +6,17 @@
...
+# Examples
+
+```julia
+T = Replace(1 => -1, 5 => -5)
+T = Replace(1 => 1.5, 5 => 5.5, 4 => true)
+```
+
+## Notes
+
+* The transform can alter the element type of columns.
```
Comment (juliohm) [inline src/transforms/replace.jl:19]:
```suggestion
```

**Review by juliohm — CHANGES_REQUESTED** [review]

File: src/transforms/identity.jl [hunk 13]
```diff
@@ -6,6 +6,10 @@
...
```
Comment (juliohm) [inline src/transforms/identity.jl:12]:
```suggestion
Identity()
```

File: src/transforms/rename.jl [hunk 14]
```diff
@@ -6,6 +6,13 @@
     Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
 
 The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+
+# Examples
+
+```julia
+T = Rename(:a => :x, :c => :y)
+T = Rename("a" => "x", "c" => "y")
```
Comment (juliohm) [inline src/transforms/rename.jl:14]:
```suggestion
Rename(:a => :x, :c => :y)
Rename("a" => "x", "c" => "y")
```

File: src/transforms/replace.jl [hunk 15]
```diff
@@ -6,6 +6,13 @@
     Replace(old₁ => new₁, old₂ => new₂, ..., oldₙ => newₙ)
 
 Replaces `oldᵢ` value with `newᵢ` value in the table.
+
+# Examples
+
+```julia
+T = Replace(1 => -1, 5 => -5)
+T = Replace(1 => 1.5, 5 => 5.5, 4 => true)
```
Comment (juliohm) [inline src/transforms/replace.jl:14]:
```suggestion
Replace(1 => -1, 5 => -5)
Replace(1 => 1.5, 5 => 5.5, 4 => true)
```

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
I did the commits myself because apparently some of my previous suggestions were ignored. I removed all ## Notes sections because they were not helpful. You add a note to a docstring when it adds new information that the user may not be aware ahead of time. For example, saying that the Identity transform preserves the schema is not very useful. Everyone knows that already and will just spend more time reading the docstring ## Notes.

File: src/transforms/identity.jl [hunk 16]
```diff
@@ -6,6 +6,10 @@
     Identity()
 
 The identity transform `x -> x`.
+
+# Examples
+
+Identity()
```
Comment (juliohm) [inline src/transforms/identity.jl:12]:
```suggestion
```julia
Identity()
```
```

File: src/transforms/identity.jl [hunk 17]
```diff
@@ -6,6 +6,10 @@
     Identity()
 
 The identity transform `x -> x`.
+
+# Examples
+```julia
+Identity()
```
Comment (juliohm) [inline src/transforms/identity.jl:12]:
```suggestion

```julia
Identity()
```
```

File: src/transforms/coalesce.jl [hunk 18]
```diff
@@ -6,6 +6,16 @@
     Coalesce(value)
 
 Replaces all missing values from the table with `value`.
+
+# Examples
+
+```julia
+T = Coalesce(0)
```
Comment (eliascarv) [inline src/transforms/coalesce.jl:13]:
```suggestion
Coalesce(0)
```

**Review by eliascarv — CHANGES_REQUESTED** [review]

**Comment by ceferisbarov** [comment]
@juliohm  I hadn't ignored the suggestions. I was going to make those changes in my computer and commit. Sorry for the confusion.

**Pull request merged**